tasks/dataverse-glassfish.yaml

```diff
@@ -1,93 +1,81 @@
+---
+# dataverse/tasks/dataverse-glassfish.yaml
+
 - name: download and unzip glassfish4
   get_url:
     url: http://dlc-cdn.sun.com/glassfish/4.1/release/glassfish-4.1.zip
     dest: /tmp
-  tags: glassfish
+    # checksum: sha1:704a90899ec5e3b5007d310b13a6001575827293 # requires ansible ≥2.0
```

Comment:
Using a checksum allows Ansible to avoid having to re-download the file on every run, which is a big time saver. However, it requires Ansible 2.0 or later, so I've held off on incorporating that.